screens/FavMoreInfo.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { StyleSheet, View, Text, Image } from 'react-native';
+import styles from '../style/base';
+import PropRow from '../components/PropRow';
+
+export default function FavMoreInfo({ route, navigation }) {
+  const { favPokemon } = route.params;
+  const { loading } = route.params;
+
+  if (loading) {
+    return (
+      <View style={styles.container}>
+        <Text>Loading...</Text>
+      </View>
+    );
+  } else {
+    return (
+      <View style={styles.container}>
+        <Text style={styles.header}>More on {favPokemon.name}!</Text>
+        <View style={stylesMore.pictures}>
+          <Image
+            style={styles.image}
+            source={{ uri: favPokemon.sprites.front_shiny }}
+          />
+
+          <Image
+            style={styles.image}
+            source={{ uri: favPokemon.sprites.back_shiny }}
+          />
+        </View>
+        <View style={styles.propCard}>
+          <PropRow left='Move:' right={favPokemon.moves[0]['move']['name']} />
+          <PropRow
+            left='Abilities:'
+            right={favPokemon.abilities.map(
+              a => !a['is_hidden'] && a['ability']['name']
+            )}
+          />
+        </View>
+      </View>
+    );
+  }
+}
+
+const stylesMore = StyleSheet.create({
```

Comment:
It would be nice to refactor the current approach as it's very confusing when you mix your "global stylesheet" (`styles`) with local one (`stylesMore`).

In React Native we usually always use only local ones (`stylesMore`) and only extract values which are reusable across the whole app (think colors) to a separate file. When we need to reuse styles we should usually think if we can reuse the whole components, not styles. Example -> instead of reusing `styles.container` you can create `FullscreenLoading` and `Fullscreen` components.

Reusing styles may lead to bugs where you e.g. reuse `propCard` in different places, then add another thing to it (e.g. padding) and end up modifying UI which shouldn't change. It's especially problematic in bigger apps. It's harder to do it with components, because you won't e.g. reuse `Fullscreen` as a wrapper for something which is not `Fullscreen`. 😛 Generally it's usually safer to copy-paste styles as it's much easier to track them then. 

Feel free to do it in a separate PR though.

BTW some people like using `styled-components` https://dev.to/amanhimself/using-styled-components-with-react-native-4k15 it's nice to know this lib, but no need to use it in this example project